### DISCUSSION TITLE
Get Rid of the For You Page

### DISCUSSION THREAD

**Discussion by LinuxDragon57** [community/community#54658] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The "For You" Page is useless on a site like Github. It is not a social media site. Classic Microsoft: Buy software then proceed to ruin that software with terrible features.

**Comment by deidyomega** [comment]
or at least put it behind following.  This isn't Instagram lol

**Comment by philiprenich** [comment]
Agreed. Hard pass on "For you". This isn't social media, I'm here to do work and this makes it more difficult.

**Comment by bn-l** [comment]
Same here. I am now missing what the people I'm following are posting. Microsoft's monetization is like death by a thousand cuts. I mean that it's fine now but you can see the trajectory and it's not a company you can trust long-term to not abuse a monopolistic position (look at the how windows has become a cynical advertising platform for the regular home user). I really wish there was a popular alternative.

**Comment by engineer-pearl** [comment]
I use Github exclusively for work at this point (I switched over to Gitlab for personal a while ago, and this account is not attached), and I can say the "For You" tab is VERY MUCH not useful. 99.9% of the time, I care about the repositories related to my company, with the other 1% being related to issues in software I am using.

It doesn't make sense for me to "discover" new repositories - I just need to find the stuff related to my job.
